bin/analyze_hoc_activity

```diff
@@ -150,6 +150,23 @@ def main()
     day += 1
   end
 
+  hoc_country_totals = {}
```

Comment:
Please add a brief header comment to the effect of "Compute the number of hoc events (the grand total and broken down by company and country)"